content/ADR-66-emotes-system-for-renderer.md

```diff
@@ -30,7 +32,7 @@ Additionally due to memory limitations, the users can now equip up to ten emotes
 ## Glossary
 
 - _Emote_: An emote is a special kind of wearable holding animations to be played by the users. The animation itself is contained in a .gltf or .glb model.
-- _Plugin_: A feature implemented in a pattern followed by the renderer: [ADR](/adr/ADR-56).
+- _Plugin_: A feature implemented in a pattern followed by the renderer: [ADR](./ADR-56-plugin-system.md).
```

Comment:
Please roll back this link, the rendered version doesn't interpret links to sources

Reply:
Oohh since I was having troubles in the editor I thought they got outdated. Reverting them!

Reply:
Done! fixed all of them